dali/kernels/slice/slice_kernel_test.h

```diff
@@ -172,29 +194,156 @@ struct SliceArgsGenerator_ExtractCenterElement {
   }
 };
 
+template <typename OutputType, int Dims>
+struct ArgsGen_BiggerThanInputSlice {
+  SliceArgs<OutputType, Dims> Get(const TensorShape<Dims>& input_shape) {
+    SliceArgs<OutputType, Dims> args;
+    for (int d = 0; d < Dims; d++) {
+      args.anchor[d] = -input_shape[d] / 2;
```

Comment:
In this case we are not hitting the input image at all as the anchor is equal -shape?

Reply:
We are hitting the image, the anchor is -shape/2 and the size is 2*shape

Reply:
You are right. I missed that.